app/src/lib/number.ts

```
@@ -0,0 +1,7 @@
+export const stringToNumber = (str: string | undefined) => {
+  if (str === undefined) return 0;
+
+  const num = Number(str);
+
+  return isNaN(num) ? 0 : num;
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)